authappliance/lib/updates.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#  copyright 2018 friedrich.weber@netknights.it
+#  License:  AGPLv3
+#  contact:  http://www.privacyidea.org
+#
+# This code is free software; you can redistribute it and/or
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or any later version.
+#
+# This code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import getopt
+import shlex
+
+from authappliance.lib.appliance import CRONTAB
+from authappliance.lib.crontabparser.cronjobparser import CronJobParser, CronJob
+
+UPDATE_COMMAND = "/usr/bin/pi-appliance-update"
```

Comment:
I just realized, the update script comes with an additional parameter
     
              -c        : clean unused packages after update.

we might add this as default. (Especially old kernel versions would fill up the system, if we do not run the apt-get autoremove).

Reply:
You are right. I added this in 0d7e150.

Reply:
so perfect.